swap/src/cli.rs

```diff
@@ -1,38 +1,74 @@
-use libp2p::core::Multiaddr;
+use libp2p::{core::Multiaddr, PeerId};
+use std::str::FromStr;
 use url::Url;
 use uuid::Uuid;
 
 #[derive(structopt::StructOpt, Debug)]
 #[structopt(name = "xmr-btc-swap", about = "Trustless XMR BTC swaps")]
 pub enum Options {
     Alice {
```

Comment:
Does this means the user needs to enter `swap alice`?
IMO, alice/bob roles are an implementation detail. Because There is only one possible direction given a role, then the role should be deduced from whether the user is sending or receiving sats (recip. piconeros)

Reply:
yes, at the moment I kept that because it would be hard to remove it. My objective is to achieve a mainnet swap to see what other obstacles we run into. Once that is done I am happy to refactor and make it more usable.

Reply:
I totally agree that we should change this to imply the role, similar to what is depicted in the mocks. I would just get it into a state where we do a swap first and then iterate on it :)